### PR TITLE
Connect csv to backend

### PR DESCRIPTION
**Pull request by marillam**
Connects import grades from file (css upload) dialog to backend.

- upload function checks response and in case of failure the dialog shows errors from server to the user
- added snack pack to inform the user about success/failure
- created own test file for the component and refactored one test from course view to it

@kiiamakir I didn't change anything about the returnImportedGrades. If something needs to be done, suggest in the review.

### CODE REVIEW

File: client/src/tests/CourseView.test.js [hunk 1]
```diff
@@ -107,31 +107,4 @@ describe('Tests for CourseView component', () => {
     expect(firstTeacherInCharge).not.toBeInTheDocument();
   });
 
-  test('CourseView should show a dialog for uploading a file when clicking on a menu button and choosing that option', async () => {
```
Comment (viivi-uhari) [inline client/src/tests/CourseView.test.js:110]: Why is this test deleted?

**Review by viivi-uhari — CHANGES_REQUESTED** [review]
Good job. The CSV import seems to work, but there's almost no checking of the file implemented in backend yet and backend isn't connected to DB yet so, changes done there might affect frontend, but for now this is a good start!

File: client/src/tests/CourseView.test.js (hunk 1 above)
Comment (kiiamakir) [reply client/src/tests/CourseView.test.js:110]: I tink this test was moved to the FileLoadDialog.test file

Comment (marillam) [reply client/src/tests/CourseView.test.js:110]: Yep, it was just moved.

**Review by kiiamakir — APPROVED** [review]

**Pull request merged**